wallet-web/components/unbond/UnbondNode.tsx

```diff
@@ -104,7 +104,7 @@ const UnbondNode = () => {
     }
 
     // we haven't clicked unbond button yet
-    if (!isLoading) {
+    if (isLoading === undefined) {
```

Comment:
I changed this slightly as `!isLoading` was preventing the `<Confirmation />` component from being displayed.

I see why before there was `isStarted` and `isFinished` states, but I think it's good to reduce the number of state variables where possible. This way we can achieve the same thing using one state variable.

Reply:
Same for the other components with this sort of structure.